## 10_fix_101779_mention_artheader_to_articleinfo_change.dpatch by Adam Di Carlo <aph@debian.org>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: The removal of ArtHeader in 4.0 is not documented in ChangeLog. This
## DP: patch fixes it.
## DP:
## DP: <URL:http://bugs.debian.org/bug=101779>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.1.2-7>

diff -urNad docbook-xml-4.5~/ChangeLog.upstream docbook-xml-4.5/ChangeLog.upstream
--- docbook-xml-4.5~/ChangeLog.upstream	2007-03-10 13:16:33.000000000 +0100
+++ docbook-xml-4.5/ChangeLog.upstream	2007-03-14 19:42:54.000000000 +0100
@@ -769,6 +769,7 @@
 - Added synopsis markup for modern programming languages (e.g, object
   oriented languages like Java, C++, and IDL)
 - Renamed DocInfo to PrefaceInfo, ChapterInfo, AppendixInfo, etc.
+  (and ArtHeader to ArticleInfo)
 - Comment was renamed Remark
 - InterfaceDefinition was removed
 
diff -urNad docbook-xml-4.5~/docbook-4.1.2/40chg.txt docbook-xml-4.5/docbook-4.1.2/40chg.txt
--- docbook-xml-4.5~/docbook-4.1.2/40chg.txt	2000-08-27 17:19:41.000000000 +0200
+++ docbook-xml-4.5/docbook-4.1.2/40chg.txt	2007-03-14 19:42:33.000000000 +0100
@@ -37,6 +37,7 @@
 - Added synopsis markup for modern programming languages (e.g, object
   oriented languages like Java, C++, and IDL)
 - Renamed DocInfo to PrefaceInfo, ChapterInfo, AppendixInfo, etc.
+  (and ArtHeader to ArticleInfo)
 - Comment was renamed Remark
 - InterfaceDefinition was removed
 
